apps/spruce/src/pages/image/tabs/EventLogTab/EventLogTab.test.tsx

```diff
@@ -0,0 +1,452 @@
+import { MockedProvider } from "@apollo/client/testing";
+import { RenderFakeToastContext } from "context/toast/__mocks__";
+import {
+  ImageEventType,
+  ImageEventsQuery,
+  ImageEventsQueryVariables,
+  ImageEventEntryAction,
+} from "gql/generated/types";
+import { IMAGE_EVENTS } from "gql/queries";
+import {
+  renderWithRouterMatch as render,
+  screen,
+  waitFor,
+  within,
+  userEvent,
+} from "test_utils";
+import { ApolloMock } from "types/gql";
+import { EventLogTab } from "./EventLogTab";
+
+const wrapper = ({ children }: { children: React.ReactNode }) => (
+  <MockedProvider mocks={[imageEventsMock]}>{children}</MockedProvider>
+);
+enum Column {
+  Name = "0",
+  Type = "1",
+  Before = "2",
+  After = "3",
+  Action = "4",
+}
```

Comment:
these should probably be numbers:
```suggestion
enum Column {
  Name = 0,
  Type = 1,
  Before = 2,
  After = 3,
  Action = 4,
}
```